### DISCUSSION TITLE
The build isn't running

### DISCUSSION THREAD

**Discussion by LeafiDev** [ppy/osu#28820] [Q&A]
I've setup the solution, I've tried to build and then run. it worked once. and never again. I have the right .net thing too
How do I get this to run?
here's it's log
(For some reason I couldn't even get it all...)

[buildIssue.txt](https://github.com/user-attachments/files/16172646/buildIssue.txt)

**Comment by bdach** [comment]
If you don't care about mobile platforms then build `osu.Desktop.slnf`. If you do care about mobile platforms, then do `dotnet workload restore`.

**Reply by LeafiDev** [reply]
I don't

**Reply by LeafiDev** [reply]
I tried to build it. It still does nothing

**Reply by LeafiDev** [reply]
I've tried both VSCode and Visual Studio

**Reply by bdach** [reply]
from the log you posted:

> C:\Users\jon\osu\osu.Android\osu.Android.csproj : error NU1202: Package HtmlAgilityPack 1.11.59 is not compatible with net8.0-android34.0 (.NETCoreApp,Version=v8.0) / android-x86. Package HtmlAgilityPack 1.11.59 does n
ot support any target frameworks. [C:\Users\jon\osu\osu.sln]

i can see you were trying to build the main sln. as stated, build the desktop slnf instead. if you tried and it still doesn't work, post an error message again. i can't magically extract it from your computer.

**Reply by LeafiDev** [reply]
loading solution provides this too
![image](https://github.com/ppy/osu/assets/83046401/4571e947-6686-4961-a4a3-8c163f28b8a8)

**Reply by LeafiDev** [reply]
running a build yields the same result. and it's 3am so I'm going to bed

**Reply by LeafiDev** [reply]
https://github.com/ppy/osu/assets/83046401/e66dda39-ca87-4be2-bbe7-8d80462674dd

this is what I see by the way
includes gamemodes, and other things

**Reply by bdach** [reply]
![1720684932](https://github.com/ppy/osu/assets/20418176/9bace23c-19bf-4cda-be50-d3ea66a52e51)

This looks as if you just don't have .NET installed (as opposed to .NET Framework)?

No idea what's going on, you'll probably have to figure it out yourself at this point.

**Reply by LeafiDev** [reply]
I went ahead and uninstalled all the existing frameworks and .net packages I had. and installed only the provided one in the README.md, currently cloning the repository.
